blitz_api/tests/tests_view_TemporaryTokenDestroy.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+from django.urls import reverse
+from django.contrib.auth import get_user_model
+
+from rest_framework.test import APIClient
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from ..models import TemporaryToken
+from ..factories import UserFactory
+
+User = get_user_model()
+
+
+class TemporaryTokenDestroyTests(APITestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+        self.user = UserFactory()
+        self.user.set_password('Test123!')
+        self.user.save()
+
+    def test_logout(self):
+        """
+        Ensure we can logout of the platform.
+        This deletes the TemporaryToken assigned to the user.
+        """
+        data = {
+            'username': self.user.username,
+            'password': 'Test123!'
+        }
+
+        response = self.client.post(reverse('token_api'), data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        token = TemporaryToken.objects.get(
+            user__username=self.user.username,
+        )
+
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+
+        response = self.client.delete(
+            reverse(
+                'authentication-detail',
+                kwargs={'pk': token.key},
```

Comment:
Move the token in the URL can be a security problem :

1. I send my token to the server in the URL
2. We have a problem (here some example):
   1. Server is overload
   2. Server fail to delete the token
   3. I have a problem to contact internet but my package is send on the local network of the office space
   4. ...
3. My token work again and people who check the network can use it

Maybe can we use the token already given in the header of the package ? :)

Reply:
Not a bad idea! 

Then call would simply be DELETE /authentication?

Reply:
Yes, and i need to do it with my current token in the header.

If my token is expired when i try to logout it will return me the 401 error and i know it's not a big deal because i'm already logout :)